lib/eigensolve_quda.cpp

```diff
@@ -668,7 +668,7 @@ namespace quda
     // 1. Take block inner product: L_i^dag * vec = A_i
     std::vector<ColorSpinorField *> left_vecs;
     left_vecs.reserve(n_defl);
-    for (int i = n_defl; i < 2 * n_defl; i++) left_vecs.push_back(evecs[i]);
+    for (int i = eig_param->n_conv; i < eig_param->n_conv + n_defl; i++) left_vecs.push_back(evecs[i]);
```

Comment:
Why is this in the no_tex branch?

Reply:
It was a quick fix that was needed after #1016 was merged.  Since this PR was almost ready this was a quick fix to add here.

Reply:
I’d generally prefer a quick hotfix pr for these things. It is orthogonal to the no Tex work.

Reply:
I figured there were enough unrelated minor fixes in this PR already that another wouldn't hurt.  Regardless, point taken.